examples/step-90/doc/intro.dox

```diff
@@ -0,0 +1,95 @@
+of $\Gamma$ in a form of quadrature points for the surface integration combined with the approximate surface normals.
+Next we distribute degrees of freedom over a thin subdomain $\Omega_h$
+that completely covers the $\Gamma_h$ and that consists of the intersected cells $\mathcal{T}_\Omega^h$,
+@f{equation*}
+  \mathcal{T}_\Omega^h = \{ T \in \mathcal{T}^{h} : T \cap \Gamma_h \neq \emptyset \}.
+@f}
+The finite element space where we want to find our numerical solution, $u_h$, is now
+@f{equation*}
+  V_h = \{ v \in C(\Omega_h) : v \in Q_p(T), \, T \in \mathcal{T}_\Omega^h \},
+@f}
+where $\Omega_h$ is the union of all intersected cells from $\bigcup_{T \in \mathcal{T}_\Omega^h} \overline{T}$.
+
+To create $V_h$, we first add an FE_Q and an
+FE_Nothing element to an hp::FECollection. We then iterate over each cell,
+$T$, and depending on whether $T$ belongs to $\mathcal{T}_\Omega^h$ or not,
+we set the active_fe_index to either 0 or 1.
+For this purpose, we will use the class NonMatching::MeshClassifier.
+
+A natural candidate for a weak formulation  involves the following bilinear forms
+@f{align*}
+  a_h(u_h, v_h) =  (\nabla_{\Gamma_h} u_h, \nabla_{\Gamma_h} v_h)_{\Gamma_h}
+                  +(u_h, v_h)_{\Gamma_h}\,,\qquad
+  L_h(v_h)      =  (f^e,v_h)_{\Gamma_h}.
+@f}
+where $f^e$ is an extension of $f$ from $\Gamma$ to $\Omega_h$.
+
+However, the so-called ''small-cut" problem may arise and one should
+introduce the stabilized version of TraceFEM: Find $u_h \in V_h$ such that
+@f{equation*}
+  a_h(u_h,v_h) + s_h(u_h, v_h) = L_h(v_h), \quad \forall v_h \in V_\Omega^h,
+@f}
+Here the normal-gradient stabilization $s_h$ involves the three-dimensional integration over whole (but intersected) cells and is given by
+@f{equation*}
+  s_h(u_h,v_h) = h^{-1}(\mathbf{n}_h\cdot\nabla u_h, \mathbf{n}_h\cdot\nabla v_h)_{\Omega_h},
+@f}
+Note that the $h^{-1}$ scaling may be relaxed for sufficiently smooth solutions such as the manufactured one, but we
+choose the strong scaling to demonstrate the worst case.
+
+<h3>Discrete Level Set Function</h3>
+In TraceFEM we construct the approximation $\Gamma_h$ using the interpolant $\psi_h$ of the exact level-set function on the bulk triangulation:
+@f{align*}
+  \Gamma_h &= \{x \in \mathbb{R}^{\text{3}} : \psi_h(x) = 0 \}.
+@f}
+The exact normal vector $\mathbf{n}$ is approximated by $\mathbf{n}_h=\nabla{}\psi_h/\|\nabla\psi_h\|$. This and the approximate quadrature rules for the integration over $\Gamma_h$ are often referred to as the "geometrical error".
+Luckily, one can show that the method converges optimally for the model problem if the same element space $V_h$ is employed for the discrete functions and  for the interpolation of  the level set function
+as if the exact domain would have been used.
+Furthermore, deal.II allows us to independently choose a more accurate geometry representation
+with a higher-order level set function, compared to the function space for solving the Laplace--Beltrami equation.
```

Comment:
Can you end this document by outlining the test case in a short section? You already state the equation and the domain here, it might be useful to also state the right hand side, and to say that because the domain (namely, the level set) is closed, it has no boundary and so no boundary conditions need to be posed.

Reply:
Since you're doing convergence tests, it may be useful to also state what the exact solution is.